### DISCUSSION TITLE
pull request requiring conflict resolving should have a nice indicator

### DISCUSSION THREAD

**Discussion by AquariusPower** [community/community#22942] [New to GitHub]
<p>this is a request for github pull requests list:</p>
<p>when there is failed travisci compilation we get a nice big red X to know we need to act.</p>
<p>when a pull request requires conflict resolving, we just gets a missing “it is working/ready” confirmation green tick.</p>
<p>for me, it is quite hard to perceive there is a missing green tick to indicate a pull request that requires conflict resolving.</p>
<p>so I would like to ask that we get a nice big yellow exclamation “!” indicating a pull request that requires conflict resolving :slight_smile:</p>
<p>another helpful thing is that my pull requests should get some kind of highlight, to be easier for me to filter them on the list, may be that item’s background could be light cyan, something like that…</p>
<p>thx vm, great coder’s site! </p>
<p>(I couldnt find a better place to post this, feel free to move it. The github pages didnt seem correct, may be I didnt understand what they are?)</p>

**Comment by ghost** [comment]
<p>GitHub Pages are a way for you to host a static site for free on GitHub. You can read more about them here <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a></p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/aquariuspower">@aquariuspower</a>,</p>
<p><em>so I would like to ask that we get a nice big yellow exclamation “!” indicating a pull request that requires conflict resolving <img src="/i/smilies/16x16_smiley-happy.png" alt="Smiley Happy" title="Smiley Happy"></em></p>
<p>Thanks for this feedback! We’re always working to improve GitHub, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>
